Sources/UIViewController+TopMostViewController.swift

```diff
@@ -26,7 +26,7 @@ extension UIViewController {
 
     /// Returns the current application's top most view controller.
     class func topMostViewController() -> UIViewController? {
-        let rootViewController = UIApplication.sharedApplication().keyWindow?.rootViewController
+        let rootViewController = UIApplication.sharedApplication().windows.first?.rootViewController
```

Comment:
@yury `UIAlertController` can become a key window. If calling `pushURL` or `presentURL` when `UIAlertController` is being shown, the view controller will be pushed or presented on `UIAlertController`.